lib/checks/color/color-contrast.json

```diff
@@ -12,6 +12,7 @@
         "imgNode": "Element's background color could not be determined because element contains an image node",
         "bgOverlap": "Element's background color could not be determined because it is overlapped by another element",
         "fgAlpha" : "Element's foreground color could not be determined because of alpha transparency",
+        "elmOutsideParent": "Element's background color could not be determined because it extends beyond a parent node",
         "default": "Unable to determine contrast ratio"
```

Comment:
I know this is not a changed line, but I have seen this message appear many, many times. What are the cases where this would still be displayed?